### PR TITLE
[IO-1072] fix for image extensions

### PR DESCRIPTION
**Pull request by Nathanjp91**
Fixes a bug on importing jpeg images in LocalDataset object

### Changelog
jpeg images now importing with localdataset object correctly

### CODE REVIEW

File: darwin/dataset/local_dataset.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from darwin.utils import SUPPORTED_IMAGE_EXTENSIONS, attempt_decode, parse_darwin_json
 
 
-class LocalDataset(object):
+class LocalDataset:
```
Comment (owencjones) [inline darwin/dataset/local_dataset.py:13]: I keep finding these in old code too

File: test.py [hunk 2]
```diff
@@ -1,25 +1,6 @@
-from typing import Optional, Union
```
Comment (owencjones) [inline test.py:1]: Feels like this has crept in in a larger PR, and probably actually needs a `git rm`?

**Review by owencjones — CHANGES_REQUESTED** [review]
Check the `test.py` file, remove or justify

File: test.py (hunk 2 above)
Comment (Nathanjp91) [reply test.py:1]: I've actually `git rm`'ed it before but it keeps coming back for some reason. It's in the gitignore

**Review by owencjones — APPROVED** [review]
LGTM

**Comment by NPatel1980** [comment]
Reviewed the fix with Nathan, QA passed

**Pull request merged**